src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -203,7 +226,7 @@ function histogramSelector(publicAPI, model) {
       }
       const mungedData = fieldNames.map(name => {
         const d = model.provider.getField(name);
-        if (!d.selectedGen) {
+        if (typeof d.selectedGen === 'undefined') {
```

Comment:
I've experimented both ways, and the use of 'typeof' makes the condition work both when the variable simply doesn't exist, and when it has been created and set to 'undefined'. I get a 'ReferenceError' thrown otherwise.